packages/malloy/src/lang/test/composite-field-usage.spec.ts

```diff
@@ -538,4 +538,81 @@ describe('composite sources', () => {
       `).toTranslate();
     });
   });
+  describe('partition composites', () => {
+    test('basic okay', () => {
+      expect(`
+        #! experimental { partition_composite { partition_field=astr partitions=[{id=ai fields=[ai]}] } }
```

Comment:
I think it should be no problem since its using annotations, but can we add a couple tests where the field names have spaces/special characters pls?